pinot-common/src/main/java/org/apache/pinot/common/utils/ZkStarter.java

```diff
@@ -191,6 +192,7 @@ public void run() {
             LOGGER.warn("Failed to connect to zk server.", e);
             throw e;
           }
+          Thread.sleep(50L);
```

Comment:
Do we need this sleep?
The CI machines running in Github Actions have very limited resource, so usually we prefer conditioned wait, instead of fixed time sleep